src/reader/ArticleReader.js

```diff
@@ -278,6 +278,20 @@ export default function ArticleReader({ api, teacherArticleID }) {
           />
         </div>
       </div>
+      <hr></hr>
+      {articleInfo.img_url && (
+        <div style={{ display: "flex", justifyContent: "center" }}>
```

Comment:
Opportunity for defining a reusable component? As opposed to inline CSS?

Reply:
This is just a center the img div, I am not sure if it extends for other things and I am not sure how useful it would be outside of this case. Do you think it makes sense?

Reply:
CenteredContent? but you're right - not the most complicated :)